<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjetHorlogeClass</class>
 <widget class="QMainWindow" name="ProjetHorlogeClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de fuseaux horaires</string>
  </property>
  <property name="windowIcon">
   <iconset resource="projethorloge.qrc">
    <normaloff>:/ProjetHorloge/clock.png</normaloff>:/ProjetHorloge/clock.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Réinitialiser</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="offsetLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Decalage manuel :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>378</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="hoursLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Édition</string>
    </property>
    <addaction name="actionAjouter"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter_2"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionAffichage_des_secondes"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAffichage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuitter_2"/>
   <addaction name="separator"/>
   <addaction name="actionAjouter"/>
   <addaction name="separator"/>
   <addaction name="actionAffichage_des_secondes"/>
  </widget>
  <action name="actionAjouter">
   <property name="icon">
    <iconset resource="projethorloge.qrc">
     <normaloff>:/ProjetHorloge/addClock.png</normaloff>:/ProjetHorloge/addClock.png</iconset>
   </property>
   <property name="text">
    <string>Ajouter une horloge</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionQuitter_2">
   <property name="icon">
    <iconset resource="projethorloge.qrc">
     <normaloff>:/ProjetHorloge/quit.png</normaloff>:/ProjetHorloge/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAffichage_des_secondes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="projethorloge.qrc">
     <normaloff>:/ProjetHorloge/secVisible.png</normaloff>:/ProjetHorloge/secVisible.png</iconset>
   </property>
   <property name="text">
    <string>Affichage des secondes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="projethorloge.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter_2</sender>
   <signal>triggered()</signal>
   <receiver>ProjetHorlogeClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAjouter</sender>
   <signal>triggered()</signal>
   <receiver>ProjetHorlogeClass</receiver>
   <slot>addHour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjetHorlogeClass</receiver>
   <slot>resetManualOffset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAffichage_des_secondes</sender>
   <signal>toggled(bool)</signal>
   <receiver>ProjetHorlogeClass</receiver>
   <slot>isSecVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addHour()</slot>
  <slot>resetManualOffset()</slot>
  <slot>isSecVisible(bool)</slot>
 </slots>
</ui>
